### ISSUE TITLE
 Item info is missing from  item_sold event

### ISSUE THREAD

**Issue by veryyoung** [ProjectOpenSea/stream-js#276]
Hello, I can't find the item info from the stream api Item Sold event
https://docs.opensea.io/reference/stream-api-event-example-payloads
node is shown on the document, but get nothing from the api

`{
    "topic": "collection:*",
    "event": "item_sold",
    "payload": {
        "event_type": "item_sold",
        "payload": {
            "event_timestamp": "2023-04-28T07:33:53.000000+00:00",
            "closing_date": "2023-04-28T07:33:53.000000+00:00",
            "is_private": false,
            "listing_date": "",
            "listing_type": "",
            "maker": {
                "address": "0x1fc10e4c74c2189531f9e7267ad5380cfa1b42d2"
            },
            "payment_token": {
                "address": "0x0000000000000000000000000000000000000000",
                "decimals": 18,
                "eth_price": 0,
                "name": "Matic",
                "Symbol": "MATIC",
                "usd_price": "1.012000000000000000"
            },
            "quantity": 1,
            "taker": {
                "address": "0x9406a44f1eeb58afe569c193c4de8b8dd1334591"
            },
            "transaction": {
                "Timestamp": "2023-04-28T07:33:53.000000+00:00",
                "hash": "0xe1ed8cdd304f88eeb30d0d5575237907920d35e35e096cc4a3f7776b1075e7eb"
            }
        },
        "sent_at": "2023-04-28T07:33:53.882537+00:00"
    },
    "ref": 0
}`

This is an example, item info is not included.

**Comment by thenerdassassin** [comment]
@veryyoung Are you seeing the item missing from some events or all events? If the sale was a bundle of items and not a single one, it would cause the payload to not include the item.

**Comment by stanjouven** [comment]
Hello,

We got the same problem as said before, we only retrieve a part of all the sold items. However, I don't understand your answer since when looking to the API documentation, there is no payload for bundle sold data but only the payload called item_sold ? 

Thank you

**Comment by ryanio** [comment]
@stanjouven bundles sold will come through item_sold but will not have the specific item data. you can probably query the tx hash separately to locate the Transfer events if you want to do item accounting for bundles sold
